components/brave_wallet/browser/solana_response_parser.h

```diff
@@ -47,6 +48,12 @@ bool ParseGetTokenAccountsByOwner(const base::Value& json_value,
 
 absl::optional<bool> ParseIsBlockhashValid(const base::Value& json_value);
 
+absl::optional<std::vector<std::tuple<std::string,  // mint
+                                      std::string,  // amount
+                                      std::string,  // ui amount
+                                      uint8_t>>>    // decimals
```

Comment:
Let's make a struct instead of tuple and put it near SolanaAccountInfo

Reply:
3956f6dd9d64f9c66df49f6f0342fc29e7191fad